iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/ProcedureManager.java

```diff
@@ -637,14 +625,32 @@ private TSStatus checkRegionMigrate(
         && ConsensusFactory.SIMPLE_CONSENSUS.equals(conf.getSchemaRegionConsensusProtocolClass())) {
       failMessage =
           "The region you are trying to migrate is using SimpleConsensus, and SimpleConsensus not supports region migration.";
-    } else if (anotherMigrateProcedure.isPresent()) {
+    } else if (0 != RegionOperationProcedure.getRegionOperations(regionGroupId)) {
+      Optional<Procedure<ConfigNodeProcedureEnv>> anotherMigrateProcedure =
```

Comment:
why need to  traversal? Cannot we use `RegionOperationProcedure.getRegionOperations(regionGroupId)` to get is directly?